templates/conf.d/nginx.conf.erb

```diff
@@ -96,6 +96,10 @@ http {
   lingering_timeout   <%= @lingering_timeout %>;
   tcp_nodelay         <%= @http_tcp_nodelay %>;
 
+<% if defined? @etag -%>
```

Comment:
`<% if @etag -%>` is fine here since etag is either `undef` or a `yes\no` string, not a boolean.

Reply:
Sure thing. I just pushed a fix for this, along with a small formatting change for the whitespace so the config file will not get an extra newline when no value is set.